src/features/share/pages/ShareAppModalVersionB.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+import styled from 'styled-components/native'
+
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiaryBlack } from 'ui/components/buttons/ButtonTertiaryBlack'
+import { AppModal } from 'ui/components/modals/AppModal'
+import { ViewGap } from 'ui/components/ViewGap/ViewGap'
+import { Share } from 'ui/svg/icons/BicolorShare'
+import { BicolorSharePhones } from 'ui/svg/icons/BicolorSharePhones'
+import { Close } from 'ui/svg/icons/Close'
+import { Invalidate } from 'ui/svg/icons/Invalidate'
+import { TypoDS } from 'ui/theme'
+import { LINE_BREAK } from 'ui/theme/constants'
+
+type Props = {
+  visible: boolean
+  close: () => void
+  share: () => void
+}
+
+export const ShareAppModalVersionB: FC<Props> = ({ visible, close, share }) => {
+  return (
+    <AppModal
+      visible={visible}
+      title="La culture, ça ce partage&nbsp;!"
```

Comment:
Il faut que tu récupères bien la phrase dans Figma (copié / collé), même si c'est une phrase courte. Ça évite ce genre de différence avec la maquette. 

```suggestion
      title="La culture, ça se partage&nbsp;!"
```